### DISCUSSION TITLE
so i signed up, and donated to the site admin, via paypal but i can't seem to get past the ..

### DISCUSSION THREAD

**Discussion by orgonesystems** [smartlike-org/smartlike#10] [Q&A] [answered]
prompt to "add funds please to account."

ive done that , i just want to try the service out now and start adding my content :)

this thing seems cool! but im stuck.  please help.  thank you 

i have a paypal transaction ID which i wont put here but can share if dev contacts me

**Comment by vadim-frolov** [comment, marked as answer]
Resolved on the same day.
